.gitignore

```diff
@@ -1,5 +1,4 @@
-### https://raw.github.com/github/gitignore/f2ce448f2ba7a092da05482ceca99209127c0884/maven.gitignore
-
+# maven
```

Comment:
Ha...pardon...wrong project. Nevermind about the copy bit. @britter just promoted CLI to git.